android/app/src/main/java/com/mattermost/rnbeta/MainApplication.java

```diff
@@ -45,24 +45,40 @@
 import com.wix.reactnativenotifications.core.AppLifecycleFacade;
 import com.wix.reactnativenotifications.core.JsIOHelper;
 
+import com.facebook.react.ReactPackage;
+import com.facebook.react.bridge.Arguments;
+import com.facebook.react.bridge.ReactContext;
+import com.facebook.react.bridge.ReactMarker;
+import com.facebook.react.bridge.ReactMarkerConstants;
+import com.facebook.react.bridge.WritableMap;
+import com.facebook.react.modules.core.DeviceEventManagerModule;
+import android.support.annotation.Nullable;
+
 import android.util.Log;
 
 public class MainApplication extends NavigationApplication implements INotificationsApplication, INotificationsDrawerApplication {
   public NotificationsLifecycleFacade notificationsLifecycleFacade;
   public Boolean sharedExtensionIsOpened = false;
   public Boolean replyFromPushNotification = false;
 
+  public static long APP_START_TIME;
```

Comment:
Do these need to be static? I assume we'll only have one instance of MainApplication at a time, but it seems like they could just be fields on the object.

Reply:
Updated as field.  I did those as static since it's used by another package (StartTime) which now I removed 'cause it's actually not needed.